src/sentry/integrations/vsts/integration.py

```diff
@@ -100,7 +97,7 @@ class VstsIntegrationProvider(IntegrationProvider):
 
     def get_pipeline_views(self):
         identity_pipeline_config = {
-            'redirect_url': absolute_uri('/extensions/vsts/setup/'),
+            'redirect_url': absolute_uri(VSTSIdentityProvider.oauth_redirect_url),
```

Comment:
Actually we want to keep this hardcoded here I think, since the `extensions/vsts/setup` is integration specific, where as identity related things not specific to integrations.